build_tools/cmake/iree_run_module_test.cmake

```diff
@@ -5,10 +5,13 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 # Utility function to return the platform name in crosscompile.
+# TODO: stop leaking CMAKE_SYSTEM_PROCESSOR values. Try to harmonize around
```

Comment:
Can you link an issue so this doesn't get lost? I guess the real issue is the usage of this in `iree_benchmark_suite_module_test` and the generated CMake that uses that? @pzread should be able to help. Should just be a matter of updating the names of the platforms in `XFAIL_PLATFORMS` and `UNSUPPORTED_PLATFORMS`, I think